backend/utils/validateEnv.js

```
@@ -0,0 +1,13 @@
+const requiredEnvVars = ["MONGO_URL", "PORT", "REDIS_URL"];
+
+function validateEnv() {
+  const missing = requiredEnvVars.filter((key) => !process.env[key]);
+
+  if (missing.length) {
+    console.error("❌ Missing required environment variables:");
+    missing.forEach((key) => console.error(`   - ${key}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)